package.json

```
@@ -1,16 +1,14 @@
 {
   "name": "@helix-design-system/helix-react",
-  "version": "1.0.0-alpha.4",
+  "version": "1.0.0-alpha.5",
   "description": "React components to wrap HelixUI custom elements",
+  "source": "src/index.mjs",
   "main": "dist/index.js",
-  "module": "dist/index.js",
-  "unpkg": "dist/index.js",
-  "files": [
-    "dist"
-  ],
+  "umd:main": "dist/index.umd.js",
+  "module": "dist/index.m.js",
   "scripts": {
     "install:clean": "rm -rf node_modules && yarn install",
-    "build": "parcel build src/index.mjs",
+    "build": "microbundle -i src/index.mjs --external react,react-dom,classnames,use-onclickoutside --name HelixReact --jsx React.createElement --jsxFragment React.Fragment --compress false --sourcemap false",
```

I for sure did not want to remove all the React from the bundles, but I think i was going overkill taking classnames, and user-onclickoutside out of the bundles. I think those have be stay in there.
```suggestion
    "build": "microbundle -i src/index.mjs --external react,react-dom --name HelixReact --jsx React.createElement --jsxFragment React.Fragment --compress false --sourcemap false",
```